modules/biobank/sql/biobankSchema.sql

```diff
@@ -0,0 +1,466 @@
+-- DROPS --
+
+/*Relational*/
+DROP TABLE IF EXISTS `biobank_container_parent`;
```

Comment:
These should be removed before the PR is merged.

Reply:
All the drops?

Reply:
Yeah all the drops. We talked a bit about this in the context of David's pull request on Monday. If someone were to run this in an upgrade patch it would delete all their data.